packages/frontend/lib/index.ts

```diff
@@ -488,6 +522,12 @@ interface TBACredentials {
     oauth_client_secret_override?: string;
 }
 
+interface TableauCredentials {
+    pat_name: string;
+    pat_secret: string;
```

Comment:
There is no way to make that less specific to Tableau? feels like it's a regular PAT credentials with a dedicated header

Reply:
I was indeed considering that angle, where we might get another provider with a similar authentication method, so as to make it less specific to Tableau. However, it appears that this is a custom `auth_mode` specific to Tableau.